tests/unit/common/automata/test_execution.h

```diff
@@ -4,12 +4,15 @@
 
 #include <vector>
 #include <string>
+#include <iostream>
 
 namespace {
 
 class TestExecution : public automata::Execution {
 protected:
     virtual automata::ExecutionControl event(automata::Event event) override {
+        std::cerr << "entering_state: " << event.entering_state << '\n';
+        std::cerr << "payload: " << (char)event.payload << '\n';
```

Comment:
Should you prefer the logging facilities of catch2?

Reply:
Ups, this wasn't suppose to be in the commit :sweat_smile:

Reply:
Didn't mean to commit this at all :grimacing:, deleting...